jupyter_client/localinterfaces.py

```diff
@@ -95,18 +97,15 @@ def _load_ips_ifconfig():
     lines = out.splitlines()
     addrs = []
     for line in lines:
-        blocks = line.lower().split()
-        if (len(blocks) >= 2) and (blocks[0] == 'inet'):
-            if  blocks[1].startswith("addr:"):
-                addrs.append(blocks[1].split(":")[1])
-            else:
-                addrs.append(blocks[1])
+        m = _ifconfig_ipv4_pat.match(line.strip())
```

Comment:
Do we want to rely on the ip address being the first match for this pattern? When I run `ifconfig`, I get the IP address, broadcast address and subnet mask on the same line - all of them in the same N.N.N.N format.